### PR TITLE
Npm audit - major update of A LOT of packages

### PR DESCRIPTION
**Pull request by stephanebisson**
**Fixed 1 low, 14 moderate and 7 high vulnerabilities.**

The remaining 10 high vulnerabilities are all caused by fabricedesre/b2gclitool. This package is old, unmaintained and only used to deploy the app to local devices for testing.

Other related changes:

- Migrated to webpack 5
- Fixed style issues in tests
- Converted `utils/confirm` to `hooks/useConfirm` to make sure all hooks are at the top-level of components. This was breaking a number of popups.

https://wikimedia.github.io/wikipedia-kaios/npm-audit/sim.html

### CODE REVIEW

**Comment by jpita** [comment]
Do we use that to deploy?

I always use the web ide

**Comment by stephanebisson** [comment]
> Do we use that to deploy?
> 
> I always use the web ide

I still use it

File: src/hooks/useConfirmDialog.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export const useConfirmDialog = () => {
+  const [showPopup] = usePopup(ConfirmDialog, { stack: true })
+  const showConfirmDialog = ({
+  	title, message, onDiscardText, onDiscard = () => {},
+  	onSubmitText, onSubmit = () => {}
+	}) => {
+		showPopup({ title, message, onDiscardText, onDiscard, onSubmitText, onSubmit })
+	}
+  return showConfirmDialog
+}
```
Comment (hueitan) [inline src/hooks/useConfirmDialog.js:13]: 👍 Nice! It also fixes the problem when I tried to update the preact to latest version due to the misuse of the hook not in the top-level component

```
hooks.module.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'context')
    at F (hooks.module.js:1)
    at usePopup (usePopup.js:5)
    at confirmDialog (confirm.js:8)
    at section (QuickFacts.js:37)
    at Object.onKeyCenter [as Enter] (useArticleLinksNavigation.js:104)
    at HTMLDocument.onKeyDown (Softkey.js:48)
```

**Review by hueitan — APPROVED** [review]
👍 great that webpack 5 works perfectly in just a few change.

**Comment by jpita** [comment]
I get a blank screen when I boot the app

https://user-images.githubusercontent.com/5482105/134010760-5d4ba628-eae4-4c93-a759-c4e75486835a.MOV

**Comment by stephanebisson** [comment]
@jpita can you connect webide and check for any console errors?

**Comment by jpita** [comment]
> @jpita can you connect webide and check for any console errors?

nothing on the console.
image
I've tried with main branch and it boots normally

**Pull request merged**